catalog-rest-service/src/main/resources/json/schema/entity/events/webhook.json

```diff
@@ -23,6 +23,20 @@
       "description": "Display Name that identifies this webhook.",
       "type": "string"
     },
+    "webhookType": {
```

Comment:
you need to add this webhookType to the existing json in the webhook table and provide the default value to generic. Otherwise, all the upgrades will be broken.

Reply:
done